sylt-parser/src/parser.rs

```diff
@@ -1086,13 +1087,37 @@ where
         let tokens = string_to_tokens(file_id, &source);
         // Parse the module.
         let (mut next, result) = module(&include, file_id, &root, &tokens);
+
         match result {
             Ok(module) => modules.push((include.clone(), module)),
             Err(mut errs) => errors.append(&mut errs),
         }
         to_visit.append(&mut next);
     }
 
+    if bundle_std {
+        modules = modules
+            .into_iter()
+            .enumerate()
+            .map(|(i, (file, mut modul))| {
+                match file {
+                    FileOrLib::File(_) => {
+                        let tokens = string_to_tokens(i, include_str!("../../std/basics.sy"));
+                        let (_, std) = module(&FileOrLib::Lib("basics"), i, &root, &tokens);
+                        let std = std.expect("Failed to parse default imports");
+                        modul.statements = [std.statements.clone(), modul.statements]
+                            .iter()
+                            .cloned()
+                            .flatten()
+                            .collect();
```

Comment:
I think this can be simplified. `modul` (which should probably be module) could probably append the `std.statements` to it, maybe even using: https://doc.rust-lang.org/std/vec/struct.Vec.html#method.append